doc/make.py

```diff
@@ -135,6 +136,12 @@ def _process_single_doc(self, single_doc):
             try:
                 obj = pandas  # noqa: F821
                 for name in single_doc.split('.'):
+                    try:
```

Comment:
cc @datapythonista single-page `pandas.io.formats.style.Styler` wasn't working earlier, since it's not available from the top-level namespace. Do we have tests for this?